libledger/Ledger.h

```diff
@@ -61,13 +61,14 @@ class Ledger : public LedgerInterface
      */
     Ledger(std::shared_ptr<dev::p2p::P2PInterface> service, dev::GROUP_ID const& _groupId,
         dev::KeyPair const& _keyPair, std::string const& _baseDir)
-      : m_service(service), m_groupId(_groupId), m_keyPair(_keyPair)
+      : m_service(service), m_groupId(_groupId)
     {
         m_param = std::make_shared<LedgerParam>();
         std::string prefix = _baseDir + "/group" + std::to_string(_groupId);
         if (_baseDir == "")
             prefix = "./group" + std::to_string(_groupId);
         m_param->setBaseDir(prefix);
+        m_keyPair = _keyPair;

```

Comment:
TODO: make keyPair to param of LedgerInterface(dev::KeyPair const& _keyPair)

Reply:
done.